public/config/configuration.json

```diff
@@ -18,5 +18,8 @@
   "SENTRY_ENABLE": "false",
   "SENTRY_DSN": "",
   "STATUS_API_URL": "https://status-api-dev.apps.silver.devops.gov.bc.ca",
-  "STATUS_API_VERSION": "/api/v1"
+  "STATUS_API_VERSION": "/api/v1",
+  "BUSINESS_SEARCH_API_URL": "https://bcregistry-dev.apigee.net/registry-search",
+  "BUSINESS_SEARCH_API_VERSION": "/api/v1",
+  "BUSINESS_API_KEY": "KHGuiTxG52ejzFZi7wu4QyD8GdVGEMAz"
```

Comment:
I think this name should be more explicit, so that it isn't confused with the same back end as, say, BUSINESS_URL.

Reply:
Just confirmed by Thor. Its a common for all business api (`you’ll use the same key for accessing the legal-api through the gateway too`) and he asked me to change the name

Reply:
ok